tests/cnf/ran/talm/talm_suite_test.go

```diff
@@ -42,9 +44,36 @@ var _ = AfterSuite(func() {
 	Expect(err).ToNot(HaveOccurred(), "Failed to delete TALM test namespace")
 })
 
-var _ = ReportAfterEach(func(report SpecReport) {
+var _ = JustAfterEach(func() {
+	var (
+		currentDir, currentFilename = path.Split(currentFile)
+		hubReportPath               = fmt.Sprintf("%shub_%s", currentDir, currentFilename)
+		spoke2ReportPath            = fmt.Sprintf("%sspoke2_%s", currentDir, currentFilename)
+		report                      = CurrentSpecReport()
+	)
+
 	reporter.ReportIfFailed(
 		report, currentFile, tsparams.ReporterNamespacesToDump, tsparams.ReporterCRDsToDump, clients.SetScheme)
+
+	if HubAPIClient != nil {
+		reporter.ReportIfFailedOnCluster(
+			RANConfig.HubKubeconfig,
+			report,
+			hubReportPath,
+			tsparams.ReporterNamespacesToDump,
+			tsparams.ReporterCRDsToDump,
+			clients.SetScheme)
+	}
+
+	if Spoke2APIClient != nil {
```

Comment:
what about spoke1?

Reply:
Spoke 1 is in KUBECONFIG so it's just the default one returned. Only Spoke 2 and Hub need to have explicit kubeconfigs set

Reply:
Oh so I guess spoke1 is covered by line 56.